src/stacked_borrows.rs

```diff
@@ -269,16 +268,24 @@ impl<'tcx> Stack {
     fn check_protector(item: &Item, tag: Option<Tag>, global: &GlobalState) -> InterpResult<'tcx> {
         if let Tag::Tagged(id) = item.tag {
             if Some(id) == global.tracked_pointer_tag {
-                throw_machine_stop!(TerminationInfo::PoppedTrackedPointerTag(item.clone()));
+                register_err(
+                    InterpError::MachineStop(Box::new(TerminationInfo::PoppedTrackedPointerTag(
```

Comment:
The naming here is really confusing now as this does *not* stop the machine.

Reply:
I know, but I have no other way to add custom errors. Should we rename it in rustc to just `Machine`?

Reply:
The non-stopping errors we are talking about here never get raised via `Err` though, right? So there's no good reason to even create an `InterpError` for them.

Reply:
oh :man_facepalming: